scripts/build-webpack.js

```diff
@@ -34,6 +34,10 @@ const compiler = webpack({
         exclude: /node_modules/,
         loader: 'babel-loader',
       },
+      {
+        test: /rx\.lite\.aggregates\.js/,
+        use: 'imports-loader?define=>false'
```

Comment:
Is it valid to have a `use` entry here? It seems to work for me too, but... `module.loaders` is apparently an alias for `module.use`. `module.use` entries must have a `loader` field. The syntax you're using here is assumed to be in a `module.rules` entry, which is how Webpack wants us to configure loaders since v2. Yet this seems to work anyway...

We should probably update webpack :/  It's difficult to even find documentation for this version.